packages/dashboard/src/components/internalDashboard/gestures/useResize.ts

```diff
@@ -1,34 +1,34 @@
 import React, { useCallback, useState } from 'react';
 import { useDispatch } from 'react-redux';
+import type { Anchor } from '~/store/actions';
 import { onResizeWidgetsAction } from '~/store/actions';
 import { toGridPosition } from '~/util/position';
-import type { Anchor } from '~/store/actions';
 import type { DashboardState } from '~/store/state';
-import type { Position, Widget } from '~/types';
+import type { Widget } from '~/types';
 import type { DragEvent } from '../../grid';
 import type { Gesture } from './types';
+import { constrainVectorToBorder } from '~/util/constrainVectorToBorder';
```

Comment:
vector is extremely generic. what this is really used to do is constraining gestures within the grid. may consider a more descriptive name rather than something overly generic

Reply:
changed it to `constrainDragEventToActualGrid`